src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/DocumentGenerator.java

```diff
@@ -63,4 +63,9 @@ public void generateForRepresentedClaim(RepresentedClaimIssuedEvent event) {
 
         publisher.publishEvent(new DocumentGeneratedEvent(event.getClaim(), event.getAuthorisation(), sealedClaim));
     }
+
+    @EventListener
+    public void generateForNonRepresentedRPA(SupportClaimIssuedEvent event) {
+        publisher.publishEvent(new DocumentGeneratedEvent(event.getClaim(), event.getAuthorisation()));
```

Comment:
Can't we not raise DocumentGeneratedEvent only? Do we require new event. If you look event listener for DocumentGeneratedEvent is Document uploader (not getting use), Staff notification (which we're OK with if staff gets 180 more claims) and RPA (which we care about).